madras-simulator/TORCS
# Custom port

This is required if multiple instances of the simulator is to be run on the same machine without port conflict. Much needed for parallel training instances.

Testing:
1. Install the scr-server integrated torcs simulator.
2. Run `$export TORCS_PORT=8899`.
3. run `$torcs`
4. You can see the torcs server waiting on the port 8899.
5. Tested with MADRaS agent madras-simulator/MADRaS@5676387 succesfully establishing a connection and sending action messages to simulator.

Testing platforms:
Ubuntu 16.04.5 LTS with g++ (Ubuntu 5.4.0-6ubuntu1\~16.04.10) 5.4.0 
Ubuntu 18.04.1 LTS with g++7 g++ (Ubuntu 7.3.0-27ubuntu1\~18.04) 7.3.0
